tasks/libs/junit_upload_core.py

```diff
@@ -27,17 +27,28 @@
 TAGS_FILE_NAME = "tags.txt"
 
 
-def add_flavor_to_junitxml(xml_path: str, flavor: AgentFlavor):
+def enrich_junitxml(xml_path: str, flavor: AgentFlavor):
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Congrats on naming and comments!

Reply:
Thanks, well `enrich_junitxml` was proposed by Alex tbh :)